api/cluster.go

```diff
@@ -52,7 +53,7 @@ func (s *Server) indexFind(ctx *middleware.Context, req models.IndexFind) {
 	for _, pattern := range req.Patterns {
 		nodes, err := s.MetricIndex.Find(req.OrgId, pattern, req.From)
 		if err != nil {
-			response.Write(ctx, response.NewError(http.StatusBadRequest, err))
+			response.Write(ctx, response.NewError(http.StatusBadRequest, err.Error()))
```

Comment:
stuff like this is not needed. Go runtime knows that an error is also a string.
https://play.golang.org/p/YZsavAxY_O